versioned_docs/version-5.x/use-link-builder.md

```diff
@@ -0,0 +1,25 @@
+---
+id: use-link-builder
+title: useLinkBuilder
+sidebar_label: useLinkBuilder
+---
+
+The `useLinkBuilder` hook let's us build a path to use for links based on a `navigate` action. It returns a function that takes `name` and `params` for the screen to navigate to and returns path based on the [`linking` options](navigation-container.md#linking).
+
+```js
+import { useLinkBuilder } from '@react-navigation/native';
+
+// ...
+
+function Home() {
+  const buildLink = useLinkBuilder();
+
+  return (
+    <a href={buildLink('Profile', { id: 'jane' })}>Go to Jane's profile</a>
+  );
+}
+```
+
+This hook is mostly useful in navigators where the navigator needs to show links to various pages in it, such as drawer and tab navigators.
+
+It's important to note that `useLinkBuilder` doesn't consider bubbling of the `navigate` action when building the link. So the screen to navigate to must be present in the navigator it's used in. For example, in the above case, the navigator containing `Home` should also contain the `Profile` screen.
```

Comment:
i think this page is a little bit confusing. we might want to lead with something saying that people probably don't need to use this (my understanding from reading it), or clarify further with a concrete example.

also, maybe we should avoid using the anchor tag above and use `Linking` instead otherwise i expect some folks to copy and paste this into their apps and be confused